build/package/generate-msi.sh

```diff
@@ -52,7 +52,5 @@ go-msi make --path "${WIX_MANIFEST_FILE}"  --msi "dist/${PACKAGE_NAME}" --versio
 go run ./tools/sign -file "dist/${PACKAGE_NAME}"
 
 if [[ "${TOOL_NAME:?}" == atlascli ]]; then
-  go run ./tools/update-chocolatey-package/update-chocolatey-package.go "${VERSION}" ./dist/"${PACKAGE_NAME}" https://fastdl.mongodb.org/mongocli/"${PACKAGE_NAME}"
-  cd ./build/package/chocolatey/temp
-  choco pack
+  go run ./tools/chocolateypkg/main.go "${VERSION}" ./dist/"${PACKAGE_NAME}" https://fastdl.mongodb.org/mongocli/"${PACKAGE_NAME}"
```

Comment:
nit: bonus points for flags
```suggestion
  go run ./tools/chocolateypkg/main.go -version "${VERSION}" -file ./dist/"${PACKAGE_NAME}" -url https://fastdl.mongodb.org/mongocli/"${PACKAGE_NAME}"
```

Reply:
Cool idea. Implemented!